db/migrate/20201021164648_add_category_to_profile.rb

```diff
@@ -0,0 +1,7 @@
+class AddCategoryToProfile < ActiveRecord::Migration[6.0]
+  def change
+    add_column :profiles, :category, :string, null: true
+    add_column :profiles, :category_code, :string, null: true
+    add_index :profiles, :category_code
```

Comment:
Do we strictly need this index? I mean, aren't we going to just guard against permitting creation of moves for a profile for a person of category A as opposed to do a bunch of reporting on the code?

Reply:
Yep, you're right, we don't strictly need this index.

It is here to help address ad hoc questions like "how many Category B prisoners have we moved?". I don't think the index size is too overbearing - it is just over 1mb for the staging database of 53,154 profiles.

Reply:
:+1: